tests/test_ssm/test_ssm_boto3.py

```diff
@@ -30,6 +30,20 @@ def test_delete_parameter():
     len(response["Parameters"]).should.equal(0)
 
 
+@mock_ssm
+def test_delete_nonexistent_parameter():
+    client = boto3.client("ssm", region_name="us-east-1")
+
+    try:
```

Comment:
Please use the _assert_raises_ paradigm for tests like this.
You can find a good example here:
https://github.com/spulec/moto/blob/master/tests/test_apigateway/test_apigateway.py#L180

Reply:
Thanks @bblommers for that feedback. I'll incorporate that now.